bpf/http_types.h

```diff
@@ -6,13 +6,21 @@
 #include "bpf_helpers.h"
 #include "http_defs.h"
 #include "pid.h"
-#include "tracing.h"
 
 #define FULL_BUF_SIZE 160 // should be enough for most URLs, we may need to extend it if not. Must be multiple of 16 for the copy to work.
 #define TRACE_BUF_SIZE 1024 // must be power of 2, we do an & to limit the buffer size
 
 #define CONN_INFO_FLAG_TRACE 0x1
 
+#define TRACE_ID_SIZE_BYTES 16
```

Comment:
Some of this stuff was just moved around, so that I can reuse helpers and structures between Go and kprobes